### ISSUE TITLE
Question: How to change metadata manager path

### ISSUE THREAD

**Issue by splattadat** [MediaBrowser/Emby#2524]
Hello,

Version 3.2.7.0 (Windows)

After running the portable for a year, I now want Emby update to be automatic. However, I must install non-portable for this feature. So I did, and copied over all my data files to the new installation location.

But I cannot change the following path...

<img src="http://i.imgur.com/U0FbFos.png">

Now, without having to delete and readd my entire libraries, where is the above path stored so that I can go in and change it? I've searched the registry, the install path, and loaded some of the DBs, and still I failed to find it. Please help.

Thanks.

**Comment by LukePulverenti** [comment]
You can't. If you want to move a file then you would need to do that in the file system, then run a library scan and Emby will pick up the changes.

**Comment by splattadat** [comment]
> then run a library scan and Emby will pick up the changes.  

Sorry, but that is not what happens.

**Comment by splattadat** [comment]
> then run a library scan and Emby will pick up the changes.  

Sorry, but that is not what happens.

The above path is stored somewhere, so that I can go in and change it, in which file is it stored on disk?

**Comment by LukePulverenti** [comment]
What path are you referring to, is this a media file?

**Comment by splattadat** [comment]
Are you answering this issue from email? I posted the screenshot above to show you directly what i am referring to, please look at the  github site directly and my post here of 6 days ago.

**Comment by LukePulverenti** [comment]
I've seen your screenshot, but the path is blurred out. That's why I asked questions about the path so that I could try and figure out what you're trying to do.

Ultimately, what you want to do is just follow these instructions:

- Is the path you want to edit a top level folder that was added in Emby library setup? If yes, then use the web interface to edit library setup as desired. That's all you need to do, and after you make changes it will kick off a library scan.

- OR, is the path a sub-folder or media file? If yes, then do whatever you need to do directly in the file system. Then once you're done, run a library scan and Emby will pick up the changes.

Thanks !

**Comment by splattadat** [comment]
>  that I could try and figure out what you're trying to do

Ok, this means I failed to express my situation, sorry. Let me try again...

I am running portable installation of Emby where **<portable location>** = `drive:\path_to_emby.windows\`

Therefore, Emby program files are stored at `<portable location>\System\` and at runtime, data files were created in `<portable location>\`

After making a couple of libraries, a folder called `<portable location>\root` was created. This location is the nub of this issue

After doing a regular windows setup, I took a copy of data files from <portable location> to the new installation location.

However, the path in the screenshot persists at `<portable location>\root\default\<library name x>` that correspond to the two libraries that I created.

I simply want to change `<portable location>` to `<base location>`. So, which file has this string value ?

**Comment by LukePulverenti** [comment]
I understand. Unfortunately at this time we do not have a documented process to surgically change the path of something. The only supported method is using the interface to re-do your library setup to the new locations.

I think this would be a good idea for us to add support for, but before we can do that we will have to test it ourselves and come up with a set of documented instructions. Thanks !

**Comment by splattadat** [comment]
> Unfortunately at this time we do not have a documented process to surgically change the path of something.

To be fair, I didn't expect you would, normally users would not surgically change a path.  I dont see this as a high priority feature to add either, therefore, i could be wait not 16 days, but i suspect months.  However, I am a dev so I have no issue doing this myself, all I need is the filename, I don't need guidance above that - or do i need to dig around in the source if you can't/won't help me.

**Comment by LukePulverenti** [comment]
You will want to change the path in the library.db database, for all affected items. so for example if this is a folder, all child items as well.

**Comment by splattadat** [comment]
Got it, thank you.
